bin/run_osa_cdci_server.py

```diff
@@ -85,28 +80,31 @@ def main(argv=None):
                                     set_by=f'command line {__file__}:{__name__}')
     use_gunicorn = args.use_gunicorn
     debug = args.debug
+    multithread = args.multithread
 
     if use_gunicorn is True:
         # let's use the bind options configuration
         dispatcher_bind_host = conf.bind_host
         dispatcher_bind_port = conf.bind_port
-        # dispatcher_url = conf.dispatcher_url
-        # port = conf.dispatcher_port
-
+        dispatcher_url = conf.dispatcher_url
+        port = conf.dispatcher_port
+        # TODO I suspect there is a bug in the gunicorn library
```

Comment:
which bug?

Reply:
While investigating on why gunicorn was not working (using the StandaloneApplication class), I kept getting the same error:

`dictionary changed size during iteration`

Did some research and one of the suggestion was to use `list` instead `items` as the second returns a view of the dictionary whereas the first one returns a whole new list. But still it didn't help.

That's why I thought of a bug within the gunicorn library, but I cannot know it for sure since then I changed implementation approach.

This should reply also to the previous question.

Reply:
You should then `copy` the values. List also contains just references. But if it's not used, can the comment reflect that?

Reply:
I can also remove it completely....